pretest.js

```diff
@@ -0,0 +1,50 @@
+var async = require('async');
+var fs = require('fs');
+var oracle = require('./')(settings);
+var settings = require('./tests-settings.js');
+
+var raw = fs.readFileSync('./test/test.sql', 'utf8');
+var statements = [];
+var stmt = '';
+// split test/test.sql file into statements, but strip any trailing ;s unless
+// they are at the end of an indented line, wich indicates a substatement.
+raw.split('\n').forEach(function(line) {
+  // test/test.sql uses "/" as dividers between related statements, so ignore
+  // them and any blank lines
+  if (/\//.test(line) || /^\s+$/.test(line)) {
```

Comment:
I thought about it, but I decided not to tie it quite that tightly to how the file looks. I'm sure this will all get cleaned up if/when someone wants to clean up the actual test sql.